src/main/java/de/tum/in/www1/artemis/service/DataExportService.java

```diff
@@ -295,7 +295,9 @@ private void addGeneralExamInformation(StudentExam studentExam, Path examWorking
         String[] headers = new String[] { "started", "testExam", "started at", "submitted", "submitted at", "working time (in minutes)", "individual end date" };
         CSVFormat csvFormat = CSVFormat.DEFAULT.builder().setHeader(headers).build();
 
-        try (final CSVPrinter printer = new CSVPrinter(Files.newBufferedWriter(examWorkingDir.resolve("exam_" + studentExam.getId() + CSV_FILE_EXTENSION)), csvFormat)) {
+        try (final var printer = new CSVPrinter(
+                Files.newBufferedWriter(examWorkingDir.resolve("exam_" + studentExam.getExam().getSanitizedExamTitle() + "_" + studentExam.getId() + CSV_FILE_EXTENSION)),
```

Comment:
This line is very long, please make the directory name an attribute (here and above).